app/components/windows/RecentEvents.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+<modal-layout
+  title="Recent Events"
```

Comment:
$t('Recent Events') here

Reply:
Thanks.  I need to start thinking about translations when I add UI now.

Reply:
Do I need to add this to one of the JSON files?

Reply:
Yeah if it is not present there